astropy/table/tests/test_operations.py

```diff
@@ -1464,18 +1473,7 @@ def test_mixin_functionality(self, mixin_cols, empty_table1, empty_table2):
         t2 = table.QTable([col2], names=["a"])
 
         # Vstack works for these classes:
-        if isinstance(
-            col1,
-            (
-                u.Quantity,
-                Time,
-                TimeDelta,
-                SkyCoord,
-                EarthLocation,
-                BaseRepresentationOrDifferential,
-                StokesCoord,
-            ),
-        ):
+        if isinstance(col1, MIXINS_WITH_FULL_MASK_SUPPORT + (StokesCoord,)):
```

Comment:
It should not be too hard to make `StokesCoord` support masking. But for another PR.